deno_dist/middleware/timing/index.ts

```diff
@@ -31,6 +31,45 @@ const getTime = (): number => {
   return Date.now()
 }
 
+/**
+ * Server-Timing Middleware middleware for Hono.
+ *
+ * @see {@link https://hono.dev/middleware/builtin/timing}
+ *
+ * @param {Partial<TimingOptions>} [config] - The options for the timing middleware.
```

Comment:
I think `Partial` is not easy to understand for users. So, I've refactored in  #2712. Can you check the latest code in `main` and fix the types?